app/services/pipeline_service/api/republish.rb

```diff
@@ -34,7 +34,7 @@ def call
       attr_accessor :range, :query
 
       def publish_all
-        self.class.models.each do |model|
```

Comment:
With this removal of this, should we also remove the `self.models` def on line 64-67?

Reply:
Nope, this transforms the model into a submission because UnitGrade is based on that.